src/Parsers/ParserSystemQuery.cpp

```diff
@@ -59,10 +59,25 @@ bool ParserSystemQuery::parseImpl(IParser::Pos & pos, ASTPtr & node, Expected &
 
         case Type::RESTART_REPLICA:
         case Type::SYNC_REPLICA:
+            if (!parseDatabaseAndTableName(pos, expected, res->database, res->table))
+                return false;
+            break;
+
+        case Type::STOP_DISTRIBUTED_SENDS:
+        case Type::START_DISTRIBUTED_SENDS:
```

Comment:
Now query `system stop distributed sends` and `system start distributed sends` doesn't work with exception
```
:) system stop distributed sends

Syntax error: failed at position 30 (end of query):

system stop distributed sends 

Expected one of: identifier, START DISTRIBUTED SENDS, ON

```